<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutBorne</class>
 <widget class="QDialog" name="AjoutBorne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjoutBorne</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant  de la borne :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdBorne">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant Voie Publique : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdVoiePublique">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom Topographique : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NomTopo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Latitude :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Latitude">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitude :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Longitude">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Ville">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrondissement :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Arrondissement">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>510</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cr√©er</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>AjoutBorne</receiver>
   <slot>valider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valider()</slot>
 </slots>
</ui>
